constants/materialStyles.ts

```diff
@@ -190,6 +190,30 @@ export const useStyles = makeStyles((theme: Theme) =>
         marginTop: '2rem',
       },
     },
+    arraignIcons: {
+      float: 'right',
```

Comment:
Floats can be a bit unwieldy and are not as widely used as they once were. Could you look into [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)? I think that you'll quickly come to like it!